Gordon360/AuthorizationFilters/StateYourBusiness.cs

```diff
@@ -421,6 +421,21 @@ private bool canUpdate(string resource)
 
                         return false;
                     }
+                case Resource.PROFILE_IMAGE:
+                    {
+                        // User is admin
+                        if (user_position == Position.GOD)
+                            return true;
+
+                        var username = (string)context.ActionArguments["username"];
+                        var profileimageservice = new ProfileImageService(new UnitOfWork());
+
+                        var isSelf = profileimageservice.Get(username).Equals(user_name);
```

Comment:
you should compare username with user_name to see whether they are the same person